conf.py

```diff
@@ -157,11 +161,39 @@
     'custom.css',
     'header.css',
     'github_issue_links.css',
-    'furo_colors.css'
+    'furo_colors.css',
+    'footer.css'
 ]
 html_css_files.extend(custom_html_css_files)
 
-html_js_files = ['header-nav.js']
+html_js_files = ['header-nav.js', 'footer.js']
 if 'github_issues' in html_context and html_context['github_issues'] and not disable_feedback_button:
     html_js_files.append('github_issue_links.js')
 html_js_files.extend(custom_html_js_files)
+
+#############################################################
+# Display the contributor
+
+def get_contributors_for_file(github_url, github_folder, pagename, page_source_suffix):
+    parsed_url = urlparse(github_url)
+    path_parts = parsed_url.path.split('/')
+    username = path_parts[1]
+    repository = path_parts[2]
+    filename = f"{pagename}{page_source_suffix}"
+    repo = Repo(".") 
+    commits = repo.iter_commits('--all', paths=filename)
```

Comment:
This works fine for this repository.
However, other repos might have the documentation files in a "docs/" subfolder, and in this case, the repo is actually on the top level and the files can be found in the corresponding subfolder.
The good thing is that we already have this subfolder defined in https://github.com/canonical/sphinx-docs-starter-pack/blob/d42ac81f5416987cbf4572c34933cadb75b85225/custom_conf.py#L98
So something like this here should work:

```
paths=html_context['github_folder'][1:]+filename
```

(The `[1:]` cuts the first character, which is a `/`.)